terraform/development/main.tf

```diff
@@ -138,6 +138,7 @@ module "simple_server_sandbox" {
   database_vpc_id               = module.simple_networking.database_vpc_id
   database_subnet_group_name    = module.simple_networking.database_subnet_group_name
   ec2_instance_type             = "t2.2xlarge"
+  database_instance_type        = "db.r4.xlarge"
```

Comment:
I believe we have a 2xlarge in production.

Reply:
Yup, in SBX, it seems like @rsanheim created a `db.r4.xlarge` – let's just stick to that for this terraform config for now?

Reply:
yea we did `xlarge` in sandbox and `2xlarge` in prod to give us plenty of headroom.

Reply:
Oh right, production is not terraform-managed. Coolbeans. 🚀